### PR TITLE
add a new file

### PR DESCRIPTION
**Pull request by dgazevedo92**

### CODE REVIEW

**Comment by sanjayankur31** [comment]
Thanks @dgazevedo92 , that looks good. Could I request you to please rename the file and change the text to use something other than "Ankur" (my name)? :) 

You can make changes and push more commits to your branch, and this pull request will continue getting updated.

File: 20210703-CNS2021/02_git/Ankur.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Ankur attended the session
```
Comment (sanjayankur31) [inline 20210703-CNS2021/02_git/Ankur.txt]: Could you update the text to use another name please?

**Review by sanjayankur31 — CHANGES_REQUESTED** [review]

File: 20210703-CNS2021/02_git/Ankur.txt (hunk 1 above)
Comment (dgazevedo92) [reply 20210703-CNS2021/02_git/Ankur.txt]: My bad @sanjayankur31 ! I made the changes. I hope it's correct now =)

**Review by sanjayankur31 — APPROVED** [review]
Looks good!

**Comment by dgazevedo92** [comment]
@sanjayankur31 , thanks for the great tutorial! I got a question and maybe you can help me. I really thought that after forking OCNS/SoftwareWG-events repository, any changes made by me would be saved only in my copy. It seems that I did something wrong because I see the changes in your repository (the file Diego.txt) but not in my copy. How can I avoid making changes to original repositories? Does this have anything to do with the difference between forking and cloning repositories?

**Comment by sanjayankur31** [comment]
The changes are in your repository but in the `ankur` branch: https://github.com/dgazevedo92/SoftwareWG-events/tree/ankur/20210703-CNS2021/02_git

To pull your `ankur` branch into your `main` branch, merge `ankur` into `main`, with something like this:

```
git checkout main
git merge ankur
```

and then push to your fork: `git push` to update your GitHub remote too.

And, because you opened a pull request here, they're also in this repository. In your pull request you asked us to merge your `ankur` branch into this repository's `main` branch.

Does that help clarify things?

**Comment by dgazevedo92** [comment]
Yes, it is much clearer now. Thanks a lot!

**Comment by sanjayankur31** [comment]
Awesome, glad that helped. Please feel free to ask more queries etc. You can use the discussions here:

https://github.com/OCNS/SoftwareWG/discussions

**Comment by dgazevedo92** [comment]
One final question. I am quite new to github, and I can find a lot of resources online for me to learn more about it. However, I feel a bit lost with so much info. Based on your experience, do you have any tips for me to improve my skills here? I have a behavioral modeling project on my computer that I want to push here. Is that a good start?

**Comment by sanjayankur31** [comment]
Yes, unless you're planning collaborating/sharing your project with others with pull requests etc. you won't really use GitHub much---the git bits covered in the tutorial would be quite enough for your daily work.

The best way to improve your skills with git is just to use it every day. When you run into a new problem, search the web---you'll almost always find a solution. If not, you can always ping the software wg and we'll help out :)

**Comment by dgazevedo92** [comment]
Awesome! Thanks a lot =D

**Pull request merged**